packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -1001,7 +1032,12 @@ describe('Collection import', function () {
       const toastElement = await browser.$(Selectors.ImportToast);
       // Check it displays that the import was aborted.
       const toastText = await toastElement.getText();
-      expect(toastText).to.include('Import aborted.');
+      try {
+        expect(toastText).to.include('Import aborted.');
+      } catch (err) {
+        console.log(toastText);
```

Comment:
The expect failure is not showing the full error message and there's a flake here. I'm trying to see what's going on.